joyfunweb/lib/joyfunweb_web/controllers/page_controller.ex

```diff
@@ -0,0 +1,72 @@
+defmodule JoyfunwebWeb.PageController do
```

Comment:
There's a fair amount of business logic in this controller. I wonder if it could have been moved into a model.